js/views/inbox_view.js

```diff
@@ -123,7 +123,8 @@
       });
 
       if (!window.storage.get('betaReleaseDisclaimerAccepted')) {
-        this.showBetaReleaseDisclaimer();
+        // Beta disclaimer disabled.
+        // this.showBetaReleaseDisclaimer();
```

Comment:
We are still in beta. why remove this?

Reply:
This is Simon's direction. He doesn't want the disclaimer anymore.